jupyterhub/templates/hub/netpol.yaml

```
@@ -40,6 +40,12 @@ spec:
     # allowed pods (hub.jupyter.org/network-access-hub) --> hub
     - ports:
         - port: http
+          # Support for Hub-managed services in k8s
+        {{- range $name, $config := .Values.hub.services }}
+        {{- if hasKey $config "k8s_managed_port" }}
+        - port: {{ get $config "k8s_managed_port" }}
+        {{- end }}
+        {{- end }}
```

I made the comment made into a helm comment to avoid confusion if inspecting the rendered template without any additional ports listed. I also suggest another syntax to accomplish the same thing as what I assume `get` does for consistency with the rest of the chart.

```suggestion
        {{- /* Also accept traffic to ports for additional services running in the hub pod /*}}
        {{- range $name, $config := .Values.hub.services }}
        {{- if hasKey $config "k8s_managed_port" }}
        - port: {{ $config.k8s_managed_port }}
        {{- end }}
        {{- end }}
```